alpaca_trade_api/rest.py

```diff
@@ -618,8 +630,10 @@ def _data_get(self,
             else:
                 path = f'/{endpoint_base}/{endpoint}'
                 data['symbols'] = ','.join(symbol_or_symbols)
+            if not endpoint_base or not symbol_or_symbols or not endpoint:
```

Comment:
This feels hacky. I suggest the following approach instead: 
- set `path` to `f'/{endpoint_base}`
- if `symbol_or_symbols` is str and not empty: add `f'/{symbol_or_symbols}'` to `path`
- if `endpoint` is not empty, add it as well